tests/respec2htmlTests/multiple-warn.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
This document seems overly complicated for something that should emit a couple of warnings. I'd suggest stripping this right down so it just emits two warnings. These could be a thing missing in the respecConfig and something missing in the document (e.g., missing heading).